smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/EnumGenerator.kt

```diff
@@ -227,19 +226,6 @@ class EnumGenerator(
      * them to camelCase after removing chars except alphanumeric, space and underscore.
      */
     fun EnumDefinition.swiftEnumCaseName(shouldBeEscaped: Boolean = true): String {
-        var enumCaseName = CaseUtils.toCamelCase(
-            name.orElseGet {
-                value
-            }.replace(Regex("[^a-zA-Z0-9_ ]"), "")
-        )
-        if (!SymbolVisitor.isValidSwiftIdentifier(enumCaseName)) {
-            enumCaseName = "_$enumCaseName"
-        }
-
-        if (shouldBeEscaped && reservedWords.contains(enumCaseName)) {
-            enumCaseName = SymbolVisitor.escapeReservedWords(enumCaseName)
-        }
-
-        return enumCaseName
```

Comment:
This logic is moved to https://github.com/awslabs/smithy-swift/pull/484/files#diff-d3747398903616500050e490227b5d5ef13f8eb5df6bdfc466974c6a4335c416R20